packages/salesforcedx-vscode-apex/src/languageServer.ts

```diff
@@ -135,6 +135,13 @@ function protocol2CodeConverter(value: string) {
 export async function createLanguageServer(
   context: vscode.ExtensionContext
 ): Promise<LanguageClient> {
+  const isInsiders: boolean = /insiders/i.test(vscode.env.appName);
```

Comment:
Decided to add this piece here since we currently have only one feature to gate based on vscode version & feature flag but let me know if we rather move this to it's own class/service.

Reply:
I think we don't need the  `: boolean` since it's a primitive. I think tslint might catch this and say it's inferable.